src/server/scan/listener/helpers/process-scan-complete.js

```diff
@@ -52,7 +52,10 @@ async function processScanComplete(server, uploadId, fileId) {
 
 function isScanningComplete(files) {
   return files.every(
-    (file) => file.fileStatus === fileStatus.rejected || file.delivered
+    (file) =>
+      file.fileStatus === fileStatus.rejected ||
+      file.fileStatus === fileStatus.complete ||
```

Comment:
Why would there be a complete but not rejected or delivered?

Reply:
This complete shouldn't be here. It will result in callbacks being made before files have been delivered